src/main/resources/application.conf

```diff
@@ -0,0 +1,22 @@
+server {
+  host = 0.0.0.0
+  port = 8080
+  ask-timeout = 5s
+}
+
+downloader {
+  max-capacity = 2
+  parallelism = 2
+  output-folder = "output/"
+  delay = 0s
+}
+
+
+csv-download-dispatcher {
```

Comment:
What is the purpose of this dispatcher? Looking at where it is used I'm not sure what was your goal of using it.

Reply:
The purpose was to use a custom dispatcher for the whole download flow (HTTP request and csv to json transformation) to avoid using the ActorSystems' default dispatcher.